src/Core/src/Platform/Android/CheckBoxExtensions.cs

```diff
@@ -26,10 +26,10 @@ public static void UpdateIsChecked(this AppCompatCheckBox platformCheckBox, IChe
 
 		public static void UpdateForeground(this AppCompatCheckBox platformCheckBox, ICheckBox check)
 		{
-			var mode = PorterDuff.Mode.SrcIn;
+			//var mode = PorterDuff.Mode.SrcIn;
 
-			CompoundButtonCompat.SetButtonTintList(platformCheckBox, platformCheckBox.GetColorStateList(check));
-			CompoundButtonCompat.SetButtonTintMode(platformCheckBox, mode);
+			//CompoundButtonCompat.SetButtonTintList(platformCheckBox, platformCheckBox.GetColorStateList(check));
+			//CompoundButtonCompat.SetButtonTintMode(platformCheckBox, mode);
```

Comment:
Can you add `Console.WriteLine()` to all of these methods? And restore the code you commented out?

Something like:
```csharp
// Method and parameters
Console.WriteLine($"{nameof(UpdateForeground)} {platformCheckBox.Handle} {check}");
// Stacktrace, so we understand who called who
Console.WriteLine($"Stacktrace: {new System.Diagnostics.StackTrace(true)}");
```

Then we could see your CWL right before the crash, maybe understanding which test called it?

My guess is it's an Android bug, and some combination of colors makes this crash.

Reply:
alright added back and pushed up